pkg/controllers/provisioningv2/rke2/provisioningcluster/template.go

```diff
@@ -407,6 +407,7 @@ func machineDeployments(cluster *rancherv1.Cluster, capiCluster *capi.Cluster, d
 					ObjectMeta: capi.ObjectMeta{
 						Labels: map[string]string{
 							capi.ClusterLabelName:           capiCluster.Name,
+							rke2.ClusterNameLabel:           capiCluster.Name,
```

Comment:
Yes, but that is already going to happen when a user upgrades to 2.6.4 because of some other changes we made. I am hoping we can get all these types of changes into 2.6.4 so it doesn't have to happen again.